### PR TITLE
New package: megazeux-2.91g

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

File: srcpkgs/megazeux/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+maintainer="Adrian Siekierka <kontakt@asie.pl>"
+license="GPL-2.0-or-later"
+homepage="http://www.digitalmzx.net/"
+wrksrc=mzx${version/.}
+_filename=${wrksrc}.tar.xz
+distfiles="http://vault.digitalmzx.net/download.php?latest=src&ver=${version}>${_filename}"
+checksum=fac0dfe1d0a6568e4338a6d9825db4d266033addfde50029feb68ad7bd5d7c4e
+
+do_configure() {
+	./config.sh --platform unix --enable-release \
```
Comment (cr6git) [inline srcpkgs/megazeux/template]: you can remove `do_configure()` and `do_build()` completely, if you set:
````
build_style=gnu-configure
configure_script="./config.sh"
configure_args="--platform unix --enable-release --bindir /usr/lib/megazeux
 --gamesdir /usr/bin --disable-updater"
````

Comment (asiekierka) [reply srcpkgs/megazeux/template]: I'm a bit mixed about that, as then build_style gives misleading information (the configure system MegaZeux uses has nothing to do with GNU configure) but sure...

Comment (cr6git) [reply srcpkgs/megazeux/template]: you could also use `build_style=configure` if you prefer...
and please remove `do_build()` too.

Comment (asiekierka) [reply srcpkgs/megazeux/template]: Done.

**Pull request merged**